jenkins/ci.opensuse.org/openstack-diskimage-builder.yaml

```diff
@@ -0,0 +1,96 @@
+- job:
+    name: 'openstack-diskimage-builder'
+    project-type: matrix
+
+    triggers:
+      - timed: '@daily'
+
+    concurent: false
+
+    build-discarder:
+    num-to-keep: 10
+    days-to-keep: -1
+
+    parameters:
+      - bool:
+          name: BOOT_DIB_IMAGE
+          default: false
+          description: Boot the generated image using QEMU/KVM.
+
+    axes:
+      - axis:
+          type: user-defined
+          name: opensuse_release
+          values:
+            - 42.1
+            - 13.1
+      - axis:
+          type: user-defined
+          name: suse_element
+          values:
+              - opensuse
+      - axis:
+          type: slave
+          name: slave
+          values:
+            - cloud-trackupstream
+
+    execution-strategy:
+      # There are a lot of mounts, loop devices etc and no mechanism
+      # to protect concurent accesses when two or more dib instances are
+      # running so run one job at a time.
+      sequential: true
+
+    builders:
+      - shell: |
+          #!/bin/bash
+          set -e
```

Comment:
the cleanvm setup works that we're starting a fresh vm from scratch, so it would be something like:

```bash

sudo /usr/local/sbin/freshvm cleanvm SLE_12_SP2
sleep 100

ssh root@cleanvm "...
```

since it is probably painful to wrap the rest of the code in a double quoted string, perhaps you can just cat <<EOF it to some file and then include that?